src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect3DatasetEngine.java

```diff
@@ -106,7 +128,7 @@ public void addData(final ReferenceContext ref, final VariantContext vc, Optiona
                         final M2ArgumentCollection.Mutect3DatasetMode mutect3DatasetMode) {
         final String refBases = ReferenceBases.annotate(ref, vc);
         final String refAllele = vc.getReference().getBaseString();
-        final String contig = vc.getContig();
+        final int contigIndex = sequenceDictionary.getSequenceIndex(vc.getContig());
```

Comment:
What happens if the sequence dictionary is null? Is that checked for somewhere? Or does GATK not allow input BAMs with no sequence dictionary?

Reply:
The sequence dictionary ultimately comes from `Mutect2::getBestAvailableSequenceDictionary()`, so it's not going to be `null` even if the BAM doesn't have one.  Would you like me to put in some `Utils.nonNull` checks somewhere?

Reply:
Nope, it sounds like `getBestAvailableSequenceDictionary()` is already handling that check, so that should be fine.